### PR TITLE
Ballistics - Fixed 6.5mm magazine config

### PR DESCRIPTION
**Pull request by ulteq**
**When merged this pull request will:**
* Add new 20rnd 6.5mm Creedmor and 6.5mm Lapua Scenar magazines
* Fix incorrect QBU-88 (DMR_07_base_F) `initSpeed`
* Fix 30rnd 6.5mm  Creedmor and 30rnd 6.5mm Lapua Scenar magazine `initSpeeds`

Closes: #4636

### CODE REVIEW

**Comment by KaiSforza** [comment]
I just have to wonder why the caseless ammo is a higher velocity than the cased ammo. Shouldn't it be the opposite?

File: addons/ballistics/CfgMagazines.hpp [hunk 1]
```diff
@@ -237,17 +240,35 @@ class CfgMagazines {
...
         ammo = "ACE_65_Creedmor_Ball";
+        initSpeed = 815;
         displayName = CSTRING(30Rnd_65_Creedmor_mag_Name);
         displayNameShort = CSTRING(30Rnd_65_Creedmor_mag_NameShort);
         descriptionShort = CSTRING(30Rnd_65_Creedmor_mag_Description);
     };
+    class ACE_20Rnd_65_Creedmor_mag: 20Rnd_650x39_Cased_Mag_F {
+        author = ECSTRING(common,ACETeam);
+        ammo = "ACE_65_Creedmor_Ball";
+        initSpeed = 808;
```
Comment (KaiSforza) [inline addons/ballistics/CfgMagazines.hpp]: Did you mean for this to be 815, and the caseless to be 808, as you did for the Scenar?

Comment (KaiSforza) [inline addons/ballistics/CfgMagazines.hpp]: Also, just looking at the data from that test that was done it seems like the base speed (if we're considering a 16" barrel the base speed for the 6.5mm cartriges) should be around 760 for the creedmoor and lapua ammo.

**Comment by ulteq** [comment]
The muzzle velocity vs. barrel length mapping happens here:
https://github.com/KaiSforza/ACE3/blob/6fddb787e3d05610f206e0c19c7330f3f866e521/addons/ballistics/CfgAmmo.hpp#L188-L189

This is our current config for 6.5 mm Creedmor:
```
        ACE_muzzleVelocities[]={750, 820, 840, 852, 860};
        ACE_barrelLengths[]={254.0, 406.4, 508.0, 609.6, 660.4};
```

We currently do not simulate caseless ammo. We assume that all vanilla 6.5mm rounds are cased.

> I just have to wonder why the caseless ammo is a higher velocity than the cased ammo.
The magazine `initSpeed` is multiplied by the weapon `initSpeed`.

**MXM:** `initSpeed: 1.01842` **30Rnd_65_Creedmor:** `initSpeed: 815` -> Muzzle velocity: 830
**DMR_07:** `initSpeed: 1.06946` **20Rnd_65_Creedmor:`** `initSpeed: 808` -> Muzzle velocity: 864

The net muzzle velocity is higher even though the magazine `initSpeed` is lower.

**Pull request merged**